napari/layers/image/image.py

```diff
@@ -454,11 +454,13 @@ def _set_view_slice(self):
                         self._top_left[d] + self._max_tile_shape,
                         1,
                     )
-                self._transform_view.translate = (
-                    self._top_left * self.scale * self._transform_view.scale
+                self._transforms['view2data'].translate = (
+                    self._top_left
+                    * self.scale
```

Comment:
I don't like that we still have a `self.scale` here, shouldn't this PR mean that all scales are tracked in transforms? This seems like information is being stored in several places (so more chance of things going wrong)

Reply:
I'm confused here also! What's `self._top_left`? Why is it different from `self.translate`?

Reply:
Top left is some extra adjustment if you have pyramid data. This conversation indicates it also needs a better name :) Another one for the brainstorming list!

Reply:
At this point `.scale` is just a convenience for `_transforms['data2world'].scale`. I think it's still nice to make that available for a user, but internally it's probably better to use the more detailed one.

`_top_left` is getting improved and could easily be renamed in #1053 where we fix a bunch of the pyramid stuff. Would be nice to get this one in without touching it and then make sure we fix that there. We can link this comment to that PR

Reply:
will add a comment about what top left is too

Reply:
added comment about top left, and put in `_transforms['data2world'].scale`. We will improve the `tile2data` / `top_left` stuff in #1053 too as soon as this merges